pages/sapin/index.tsx

```diff
@@ -18,11 +19,16 @@ export default function Home({ color, products }: Props) {
   return (
     <Page>
       <Text variant="h2" className="mb-6" style={{ color }}>
-        Home page
+        SAPIN
       </Text>
       <Text className="text-lg mb-4">
-        You&apos;re currently visiting the <b>brand {brand.toUpperCase()}</b> {products.map(({name}) => name).join(', ')} website.
+        You&apos;re currently visiting the <b>brand {brand.toUpperCase()}</b> {
+          products.map(({name}) => name).join(', ')
+        } website.
       </Text>
+      <div>
+        { products.map((product) => <Product product={product}></Product>) }
```

Comment:
We could also do this like:
```suggestion
        { products.map((product) => <Product {...product}></Product>) }
```

This would require changes to `Product` component also like:
```jsx
const Product = ({ id, thumbnail, cost }) => {
```